### DISCUSSION TITLE
How to send a POST request using RedirectResponse?

### DISCUSSION THREAD

**Discussion by dhruv-ahuja** [encode/starlette#1773] [Q&A] [answered]
Hi, can someone help me understand how I can send a `post` request from a `get` endpoint using `RedirectResponse`? Or is that not possible at all? I know that we can do the inverse by changing the status code to a `303` instead of the default `307`. 
Any help would be appreciated, thanks!

**Comment by Kludex** [comment, marked as answer]
You can't.
